apps/meteor/server/services/federation/service.ts

```diff
@@ -317,4 +317,8 @@ export class FederationService extends AbstractBaseFederationService implements
 			FederationRoomSenderConverter.toCreateDirectMessageRoomDto(internalInviterId, internalRoomId, externalInviteeId),
 		);
 	}
+
+	public async verifyMatrixId(matrixId: string): Promise<string> {
```

Comment:
Wouldn't be better to put this "common" method between CE and EE in the abstract class, so we can use the same for both?